tests/integration/test_wpk/test_wpk_manager.py

```diff
@@ -35,6 +37,19 @@
 global valid_sha1_list
 valid_sha1_list = {}
 
+if not global_parameters.wpk_version:
+    raise Exception("The WPK package version must be defined by parameter. See README.md")
+version_to_upgrade = global_parameters.wpk_version[0]
+
+def get_current_version():
+    if platform.system() == 'Linux':
+        config_file_path = os.path.join(WAZUH_PATH, 'etc', 'ossec-init.conf')
+        _config = ConfigObj(config_file_path)
+        return _config['VERSION']
+    return None
```

Comment:
```suggestion
```
`None` is the default return value of Python's functions.

Reply:
Fixed